src/network/networkd-network.h

```diff
@@ -297,6 +297,7 @@ struct Network {
         uint32_t cost;
         uint16_t priority;
         MulticastRouter multicast_router;
+        int locked;
```

Comment:
For a new element, please prefix with `bridge_`, like other elements e.g. `dhcp_xyz`.
Yes, most of the above elements are not prefixed, but for new one, please.